app/assets/javascripts/event_staff.js

```diff
@@ -1,23 +1,80 @@
-$(document).ready(function(){
-  $(".add_staff_field").autocomplete({
-    delay: 500,
-    minLength: 4,
-    source: "/lookup_staffs",
-    select: function(event, ui){
-      var staffId = ui.item.data;
-      if ($(".event_staffs").length > 0){
-        $.get("/user_info/" + staffId, function(data){
-          $(".event_staffs").append(data);
-          $(".add_staff_field").val("");
-        });
-      }
-      else {
-          $("#new_staff").val(staffId);
-      }
-    }
-  });
 
-  $(".event_staffs").on("click", ".remove_staff", function(){
-    $(this).parents(".chip").hide("slow", function() { $(this).remove(); });
+  $(document).ready(function(){
+
+    var from_page = document.getElementsByClassName("chip");
+    $(".add_staff_field").autocomplete({
+        delay:500,
+        minLength: 4,
+        source: "/lookup_staffs",
+        select:function(event, ui){
+          var staffId = ui.item.data;
+          $("#staff_id").val(staffId);
+        }
+    });
+
+
+      $(".event_staffs").on("click", ".remove_staff", function(){
+        $(this).parents(".chip").hide("slow", function() { $(this).remove(); });
+      });
+
+
+      $("#add_staff").on("click", function(){
+        var check_field = $("#staff_field_id").val()
+
+        if (check_field != ""){
+          var staffId = $("#staff_id").val();
+          var staffRole = $("[name='role']").val();
+          var chips_div = $('.chip');
```

Comment:
Identifier 'chips_div' is not in camel case.<br>Mixed double and single quotes.<br>'chips_div' is defined but never used.